app/split_integer.py

```diff
@@ -1,7 +1,13 @@
+
 def split_integer(value: int, number_of_parts: int) -> list:
     parts = []
     for parts_left in range(number_of_parts, 0, -1):
         next_number = value // parts_left
         parts.append(value // parts_left)
+        print(parts_left, parts, value)
         value -= next_number
+        # print(value)
     return parts
+
+
+split_integer(32, 33)
```

Comment:
The test case `split_integer(32, 33)` will not work as expected because the number of parts exceeds the value. Consider adding logic to handle such cases.